packages/lib/gpcircuits/scripts/gen-circuit-parameters.ts

```diff
@@ -0,0 +1,95 @@
+import circomkitJson from "../circomkit.json";
+import { Circomkit } from "circomkit";
+import * as fs from "fs/promises";
+import { existsSync as fsExists } from "fs";
+import * as path from "path";
+import {
+  ProtoPODGPC,
+  ProtoPODGPCParameters,
+  PROTO_POD_GPC_PUBLIC_INPUT_NAMES
+} from "../src/proto-pod-gpc";
+import { batchPromise, clearDir, maxParallelPromises } from "../src/util";
+
+// Circuit parameters used to generate artifacts.
```

Comment:
This is the root source of truth for these parameters, right?  Worth calling that out in the comment and/or the name since it took me a while to read enough to understand which pieces where generated and which were original.